addons/mail/static/src/components/thread_view_topbar/thread_view_topbar.scss

```diff
@@ -92,4 +94,13 @@
 .o_ThreadViewTopbar_threadName {
     font-size: $font-size-lg;
     font-weight: $font-weight-bold;
+
+    &:not(.o-over) {
+        border-color: transparent; // presence of border even if invisible to prevent flicker
+    }
+
+    &.o-over {
+        background-color: $white;
+        border-color: $border-color;
+    }
```

Comment:
For modifiers, make it clear the boolean state and the source from model on the name.
Here it's the `isOverThreadName` of `threadView`.

I have no clue what's it means being "over thread name", it's quite strange wording actually. For me, something "being over" usually means it's terminated. Of course, this is not what's the meaning here, I suppose it's positional adjective. This is very context-heavy, so you can't just say "over" in this context.

Reply:
I'll rename the modifier `isMouseOverThreadName` too.